booklets_text/2014.txt

```diff
@@ -518,6 +518,7 @@ The final tool is called maK_it and was created on the CentOS linux distribution
 maintain root access to a Linux host computer in a stealthy way once installed. It can be used for
 information reconnaissance and remote hidden access to a compromised machine. It employs
 multiple techniques to avoid detection.
+Project Area
```

Comment:
Add this so that description can be extracted successfully. However project area and the rest of the fields are still left out.